spec.html

```diff
@@ -28586,29 +28371,14 @@ <h1>Math.tanh ( _x_ )</h1>
       <emu-clause id="sec-math.trunc">
         <h1>Math.trunc ( _x_ )</h1>
         <p>Returns the integral part of the number _x_, removing any fractional digits. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is greater than 0 but less than 1, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is less than 0 but greater than -1, the result is *-0*.
-          </li>
-        </ul>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *-0*, _n_ is *+0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. If _n_ &gt; 0 and _n_ &lt; 1, return *+0*.
+          1. If _n_ &lt; 0 and _n_ &lt;= -1, return *-0*.
+          1. Let _r_ be _n_ modulo 1.
+          1. Return _n_ - _r_.
```

Comment:
This doesn't work for negative `x`, right? I think you'll have to say something like

```suggestion
          1. If n &gt; 0, return _n_ - _r_; otherwise, return _n_ + (1 - _r_).
```

Reply:
Yep!